### PR TITLE
Remove the HiFiAC codec

### PR DESCRIPTION
**Pull request by ksuprynowicz**
(I copied Dale's PR from Vircadia and updated it to newest master branch)
This is a proprietary codec and it's uncertain who can use it, and
under what conditions. At this point, Opus is stable and a suitable
replacement.

### CODE REVIEW

File: domain-server/resources/describe-settings.json [hunk 1]
```diff
@@ -1343,8 +1343,8 @@
           "name": "codec_preference_order",
           "label": "Audio Codec Preference Order",
           "help": "List of codec names in order of preferred usage",
-          "placeholder": "opus, hifiAC, zlib, pcm",
-          "default": "opus,hifiAC,zlib,pcm",
+          "placeholder": "opus,  zlib, pcm",
```
Comment (JulianGro) [inline domain-server/resources/describe-settings.json:1346]:
```suggestion
          "placeholder": "opus, zlib, pcm",
```

**Comment by ksuprynowicz** [comment]
Right now it fails cmake configure step:
```
CMake Error at plugins/CMakeLists.txt:55 (add_subdirectory):
  add_subdirectory given source "hifiCodec" which is not an existing
  directory.
```

**Comment by ksuprynowicz** [comment]
I fixed the cmake error

**Comment by JulianGro** [comment]
There is still a merge conflict. Just FYI.

**Review by daleglass — APPROVED** [review]

**Pull request merged**